src/scss/yellow/components/_header-banner.scss

```diff
@@ -27,15 +28,15 @@ $box-shadow: rgba($color-interface-content-extra-dark, .04);
   }
 
   &--border {
-    border-bottom: 1px $border-style-default $color-interface-shade-dark;
```

Comment:
Will not have border?

If not, it would make more sense to rename the modifier.